hieradata/site/dev/role/graylog.yaml

```diff
@@ -0,0 +1,4 @@
+---
+classes:
+  - "profile::core::common"
+  - "role_graylog"
```

Comment:
Are we still deploying graylog via Puppet in the dev environment?

Reply:
deleted